libraries/incubator/ORM/Entity/EntityBuilder.php

```diff
@@ -385,11 +382,12 @@ public function handleStorage(Element $storage)
 					{
 						case 'csv':
 							$handler = '\Joomla\ORM\Storage\CsvProvider';
-							$param   = $parts[1];
+							break;
+						case 'orm':
+							$handler = '\Joomla\ORM\Storage\Doctrine\DoctrineProvider';
```

Comment:
I would really leave it as it is as it is more flexible and making the changes afterwards would be much more efforts. I'm sure there will be a need to pass more parameters to doctrine which can not be configured trough an url later on, for example dirver specific settings as described here http://doctrine-orm.readthedocs.io/projects/doctrine-dbal/en/latest/reference/configuration.html#pdo-sqlite.